spec/brcobranca/base_spec.rb

```diff
@@ -179,6 +179,18 @@ module Boleto #:nodoc:[all]
         expect(boleto_novo.respond_to?(:lote)).to be_truthy
         expect(boleto_novo.respond_to?(:to)).to be_truthy
       end
+
+      context 'Validações' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.